demo/image_demo.py

```diff
@@ -12,8 +14,15 @@ def main():
         '--device', default='cuda:0', help='Device used for inference')
     parser.add_argument(
         '--score-thr', type=float, default=0.3, help='bbox score threshold')
+    parser.add_argument(
+        '--async-infer',
```

Comment:
--async shoud be OK

Reply:
if use --async, `args.async` will show python keyword conflicts in IDE. Although it will not an error, it is not very good.